tests/integration/ceph_object_test.go

```diff
@@ -184,7 +184,8 @@ func testObjectStoreOperations(s *suite.Suite, helper *clients.TestClient, k8sh
 		assert.Nil(t, cobErr)
 		cobcErr := helper.BucketClient.CreateObc(obcName, bucketStorageClassName, bucketname, maxObject, true)
 		assert.Nil(t, cobcErr)
-
+		logger.Info("waiting 10 more seconds before we check OBCs are bound")
+		time.Sleep(10 * time.Second)
 		created := utils.Retry(12, 2*time.Second, "OBC is created", func() bool {
```

Comment:
Instead of adding a sleep on the line above, can we increase the retry count here?
```suggestion
		created := utils.Retry(20, 2*time.Second, "OBC is created", func() bool {
```